content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -215,6 +219,50 @@ malicious) damage to the host. The LocalSystem service account has the highest l
 of privilege of the three and should be used only if absolutely necessary. Where possible,
 use the LocalService service account as it is the least privileged of the three options.
 
+### Local accounts {#local-accounts}
+
+HostProcess containers can also run as local user accounts which allows for node operators to give
+fine-grained access to workloads.
+
+To run HostProcess containers as a local user; A local usergroup must first be created on the node
+and the name of that local usergroup must be specified in the `runAsUserName` field in the deployment.
+This will cause an new ephemeral local user account to be created, joined to the specified usergroup,
+and used by the container. This provides a number a benefits including eliminating the need to manage
+passwords for local user accounts.
+
+{{< note >}}
+Running HostProcess containers as local user accounts requires containerd v1.7+
+{{< /note >}}
+
+Example:
+
+1. Create a local user group on the node (this can be done in another HostProcess container).
+
+    ```cmd
+    net localgroup hpc-localgroup /add
+    ```
+
+1. Grant access to desired resources on the node to the local usergroup.
+    This can be done with tools like [icacls](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/icacls).
```

Comment:
```suggestion
   This can be done with tools like [icacls](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/icacls).
```